Circuit_Playground_Express_cpx_Library_Examples/cpx_slide_switch_red_led.py

```diff
@@ -0,0 +1,11 @@
+"""This example uses the slide switch to control the little red LED."""
+from adafruit_circuitplayground.express import cpx
+
+while True:
+    if cpx.switch:
+        cpx.red_led = True
+    else:
+        cpx.red_led = False
+
+# Can also be written as:
```

Comment:
This isn't needed if there is going to be a separate example that does the same thing